### ISSUE TITLE
Headers getting removed

### ISSUE THREAD

**Issue by connordodge** [ivan-lednev/obsidian-day-planner#31]
I took the content from your day-planner-example.md file and pasted it into my day planner section.
```
This is my plan for the day broken into 3 main sections:
1. Morning Prep
2. Reading
3. Afternoon Review

### Morning Prep

This is where I get ready for work and do my usual prep.

- [ ] 09:30 Setup for work
- [ ] 09:45 Review notes from yesterday
- [ ] 10:30 Create new notes for #article review
- [ ] 11:30 BREAK

### Reading

A section of the day dedicated to reading:

1. Articles.
2. Book chapters assigned for the day.
3. Re-reading past notes.
   
- [ ] 12:00 Reading
  - [ ] Article 1
  - [ ] Article 2
  - [ ] Article notes review
- [ ] 12:25 BREAK
- [ ] 12:30 Reading
- [ ] 14:00 BREAK

### Afternoon Review

I use this time to review what I have done earlier in the day and complete any tasks to prepare for the next day.

- [ ] 15:00 Review notes and update daily note [[20201103]]
- [ ] 15:45 Walk
- [ ] 16:30 Reading
- [ ] 17:20 Prep for tomorrow's meetings
- [ ] 18:00 END
```

It removed all of the headers and extra text outside of the tasks and left me with this

```
- [ ] 09:30 Setup for work
- [ ] 09:45 Review notes from yesterday
- [ ] 10:30 Create new notes for #article review
- [ ] 11:30 BREAK
- [ ] 12:00 Reading
  - [ ] Article 1
  - [ ] Article 2
  - [ ] Article notes review
- [ ] 12:25 BREAK
- [ ] 12:30 Reading
- [ ] 14:00 BREAK
- [ ] 15:00 Review notes and update daily note [[20201103]]
- [ ] 15:45 Walk
- [ ] 16:30 Reading
- [ ] 17:20 Prep for tomorrow's meetings
- [ ] 18:00 END
```

**Comment by lynchjames** [comment]
Are you using the latest version (0.5.3)?

**Comment by connordodge** [comment]
@lynchjames I am. I just checked though, and obsidian auto updated to the latest version (9.15). I switched it back to 9.10 and it is working.

**Comment by lynchjames** [comment]
I have had some intermittent issues with the a new plugin version not loading correctly and having to either reload or restart Obsidian. It could be that it was the restarting of Obsidian that resolved your issue.

Version 0.5.3 of the plugin should work with the latest public version of the plugin (0.9.15).

**Comment by lynchjames** [comment]
Hi @connordodge. Were you able to get the plugin working on the latest version of plugin and Obsidian?

**Comment by manuelalonso** [comment]
I am also having an issue with Dayly Planner removing, not headers, but tasks themselves. This happens at fixed intervals of 1 minute or so.  Al new tasks added get erased and the end result is a version of the Planner that I do not now where it is getting it!

**Comment by ivan-lednev** [comment]
This should be fixed with 0.7.0.
